components/layers/hs.source.Wfs.js

```diff
@@ -100,6 +100,5 @@ export default function (options) {
     projection: options.projection,
     strategy: loadingstrategy.bbox,
   });
-  src.defOptions = options;
```

Comment:
This change just reminded me that this file is not used anymore. Its all handled inside add.layers.wfs

Reply:
I was suspecting that. Should I delete it?